evalml/tests/demo_tests/test_datasets.py

```diff
@@ -1,28 +1,47 @@
+import urllib
+
 import pandas as pd
+import pytest
 
 from evalml import demos
 
 
-def test_fraud():
-    X, y = demos.load_fraud()
+@pytest.fixture(autouse=True)
+def set_testing_headers():
+    opener = urllib.request.build_opener()
+    opener.addheaders = [("Testing", "True")]
+    urllib.request.install_opener(opener)
+
+
+@pytest.fixture(autouse=True)
```

Comment:
Can we run this fixture in session scope? So that it is only run once per pytest run? This will minimize the chance of network flakes

https://docs.pytest.org/en/6.2.x/fixture.html#scope-sharing-fixtures-across-classes-modules-packages-or-session